packages/remix/src/generators/utils/update-dependencies.ts

```diff
@@ -0,0 +1,33 @@
+import { type Tree, addDependenciesToPackageJson } from '@nx/devkit';
+import {
+  eslintVersion,
+  isbotVersion,
+  reactDomVersion,
+  reactVersion,
+  remixVersion,
+  typescriptVersion,
+  typesReactDomVersion,
+  typesReactVersion,
+} from '../../utils/versions';
+
+export function updateDependencies(tree: Tree) {
+  return addDependenciesToPackageJson(
+    tree,
+    {
+      '@remix-run/node': remixVersion,
+      '@remix-run/react': remixVersion,
+      '@remix-run/serve': remixVersion,
+      isbot: isbotVersion,
+      react: reactVersion,
+      'react-dom': reactDomVersion,
+    },
+    {
+      '@remix-run/dev': remixVersion,
```

Comment:
We could leave this one out since it should be installed by the `init` generator.